org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -141,4 +141,116 @@
       }
     }
   }
+
+  &.page-civicrm-vacancy-publiclisting table {
+    @extend %crm-table;
+    box-shadow: $box-shadow;
+
+    tr {
+      &:first-child {
+        th {
+          &:first-child {
+            border-top-left-radius: 2px;
+          }
+
+          &:last-child {
+            border-top-right-radius: 2px;
+          }
+        }
+      }
+
+      &:last-child {
+        td {
+          &:first-child {
+            border-bottom-left-radius: 2px;
+          }
+
+          &:last-child {
+            border-bottom-right-radius: 2px;
+          }
+        }
+      }
+
+      td {
+        color: $crm-copy;
+      }
+    }
+
+    &.vacancy-popup {
+      margin-bottom: 20px;
+    }
+
+    &.selector {
+      margin-top: 20px;
+    }
+  }
+
+  .crm-container.ui-dialog {
```

Comment:
Is this dialog a generic civicrm dialog? If yes, do you think it could deserve a standalone partials with generic rules (right now this styling is applied only for this page), so that modals (i assume this is for a modal) gets automatically styled in every page?